assets/src/edit-story/components/form/color/colorPreview.js

```diff
@@ -161,36 +144,55 @@ function ColorPreview({
     previewText,
   ]);
 
-  // Always hide color picker on unmount
-  useUnmount(hideSidebar);
-
-  if (isEditable) {
-    // If editable, only the visual preview component is a button
-    // And the text is an input field
-    return (
-      <Preview ref={ref}>
-        <VisualPreview role="status" style={previewStyle} {...buttonProps} />
-        <TextualInput
-          aria-label={`${inputLabel}: ${label}`}
-          value={hexInputValue ?? ''}
-          onChange={handleInputChange}
-          onBlur={handleInputBlur}
-        />
-      </Preview>
-    );
-  }
+  // Always hide color picker on unmount - note the double arrows
+  useUnmount(() => () => setPickerOpen(false));
+
+  const onClose = useCallback(() => setPickerOpen(false), []);
+  const spacing = useMemo(() => ({ x: 20 }), []);
```

Comment:
What's this for?

Reply:
Hmm my understanding is that it prevents needless rerenders on the spacing object, but I might be wrong?